modules/core/src/main/scala/org/tessellation/domain/cluster/services/L0Cluster.scala

```diff
@@ -0,0 +1,7 @@
+package org.tessellation.domain.cluster.services
+
+import org.tessellation.schema.peer.L0Peer
+
+trait L0Cluster[F[_]] {
+  def getL0Peers: F[Set[L0Peer]]
+}
```

Comment:
L0 cluster is native in this layer/module, so `clusterStorage.getPeers` should be used

Reply:
I could use `cluster.info` in the routes class and do the adjusting to `L0Peer` there if that's what you mean? That would indeed make more sense than creating additional `L0Cluster`.

Reply:
I could use `cluster.info` in the routes class and do the adjusting to `L0Peer` there if that's what you mean? That would indeed make more sense than creating additional `L0Cluster`.